### PR TITLE
Add section to explain liquidity tier model

### PR DESCRIPTION
**Pull request by CoinGame**
Any thoughts on the wording? I pretty much pulled it directly from the link on the page with some modification to tier one. Has it changed, or is this all still accurate?

### CODE REVIEW

File: _kb/liquidity-tiers.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## NuBits Liquidity Tiers
+
+### Tier 1
+
+This liquidity is immediately available, being on the order book. It is used by NuBot to maitain the 1$ peg on exchanges.
+
+### Tier 2
+
+This liquidity sits on exchanges but is not placed on order books and can be promoted to the order book (Tier 1) in a couple seconds.
```
Comment (desrever-nu) [inline _kb/liquidity-tiers.md]: Ok, here we need some thoughts on tier1 and tier2.

Tier1 is on the orderbook AT the best price (1$ +- fees and spread) . This is "the wall" .
Tier2 can be sitting on the account OR placed on the orderbook at premium prices .

**Comment by desrever-nu** [comment]
Very well explained, I just added a one line comment because we need to rephrase Tier2 explanation, as sometimes Tier2 is on the orderbook, just at different prices

**Comment by CoinGame** [comment]
@desrever-nu I've updated the wording. Does that fit a bit better?

**Comment by desrever-nu** [comment]
Yepppa!

**Pull request merged**